content/docs/iac/concepts/resources/providers.md

```diff
@@ -91,17 +91,40 @@ Installing package 'random'...
 If you are tracking a package in the project file and installing via `pulumi install`, be sure to remove any generated SDK files from version control and `.gitignore` the SDK directory or the generated files will still be under version control!
 {{% /notes %}}
 
-## Default Provider Configuration
+## Default and Explicit Providers
 
-By default, each provider uses its package’s global configuration settings, which are controlled by your stack’s configuration. You can set information such as your cloud provider credentials with environment variables and configuration files. If you store this data in standard locations, Pulumi knows how to retrieve them.
+Within a Pulumi program, there are two types of providers you can use to declare resources:
 
-For example, suppose you run this CLI command:
+- **Default providers** are not declared in your Pulumi program, and use global configuration settings. Resources created with a default provider do not need the [`provider` option](/docs/iac/concepts/options/provider/) set in the [resource's options parameter](/docs/iac/concepts/options/). This is the most common way to declare Pulumi resources.
```

Comment:
The "most common way" sentence hangs in the air a bit. Why is most common?

Reply:
Changed to "simple", which still might hang in the air, but less so. I figure with the table that breaks things down, we provide enough context for "simple" to work here.